libs/velo-external-db-core/src/service/data.spec.ts

```diff
@@ -70,13 +71,13 @@ describe('Data Service', () => {
     test('bulk insert will insert data into db', async() => {
         driver.expectInsertFor(ctx.entities, ctx.collectionName, ctx.defaultProjection)
 
-        return expect(env.dataService.bulkInsert(ctx.collectionName, ctx.entities, ctx.defaultProjection)).resolves.toEqual({ items: ctx.entities })
+        return expect(env.dataService.bulkInsert(ctx.collectionName, ctx.entities, ctx.defaultProjection)).resolves.toEqual({ items: ctx.entities.map(asWixDataItem) })
```

Comment:
actually here I think its fine, because we are in kind of unit test, and we are testing the data service, so one of the service job is to return items after `asWixData` (I don't really care what `asWixData` does as long as it tested somewhere else)

Reply:
Yup, I am checking this function (asWixData) in `data_utils.spec`